src/app/master-view/master-view.component.ts

```diff
@@ -34,21 +36,29 @@ export class MasterViewComponent implements OnInit {
 
   public selectCustomer(customerID: string) {
     this.selectedOrderId = undefined;
-    this.northwindJasonApiCustomerOrdersCustomerId = [];
-    this.northwindJasonApiCustomerOrderDetailsOrderId = [];
+    // this.northwindJasonApiCustomerOrdersCustomerId = [];
+    // this.northwindJasonApiCustomerOrderDetailsOrderId = [];
```

Comment:
Nice using the `isLoading` flag. I wasn't aware of it.
We still need to clean order details though, otherwise it shows outdated data when changing Customer.
Or.. always initialize it with the first loaded order.